homework/homework_part_2/javascript/app.js

```diff
@@ -0,0 +1,133 @@
+console.log('Welcome');
+
+// Question 1. The difference between an Interpolation is using 
+// variables in a console.log vs not using them.
+
+// Concatenation - console.log('Hello' + ' Good' + ' Morning');
+	
+//Interpolation 	
+// const word = "wonderful";
+// console.log('What a ' + word + ' day');
+
+//Question 2. DRY stand for "Don't Repeat Yourself".
+//We should pay attention to it to prevent clutter
+//and have nicer code. The tools we learned to write DRY code
+//are creating loops as opposed to repeating a console.log
+
+
+//Quesstion 3.  The difference bettween declaring a variable
+//and assigning a value to a variable is as simple as
+//const variable; (declaring a variable)
```

Comment:
yes -- except with const you must initialize with a value. (try it, you will get: Uncaught SyntaxError: Missing initializer in const declaration)
however, you can declare a variable with let without assigning:
```
let x;
```